### PR TITLE
Gnuykeaj theme

### PR DESCRIPTION
**Pull request by andyklimczak**
Super minimal theme that works with git, and a backwards name 

:koala:

### CODE REVIEW

File: themes/gnuykeaj/fish_prompt.fish [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# name: gnuykeaj
+# ---------------
+# Based on clearance, which is based off idan.
```
Comment (bpinto) [inline themes/gnuykeaj/fish_prompt.fish]: This description is more likely to be read if it's on the README file. I'd move this description block to README.

**Comment by bpinto** [comment]
You have added the screenshot to git in one of the commits, this increases the git repo considerably. Would you squash these commits so that git no longer has any traces of the screenshot? Thanks.

**Comment by andyklimczak** [comment]
Sure, I'll make it clean and do a new pr.

**Comment by bpinto** [comment]
:+1:

**Pull request closed without merging**